### PR TITLE
Fix for requestor's time window in the 'force subtask results' should…

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent/issues/105.

### CODE REVIEW

File: concent_api/core/views.py [hunk 1]
```diff
@@ -457,28 +457,39 @@ def handle_send_force_subtask_results_results_response(request, client_message):
...
+
+    assert 1 <= force_subtask_results.count() <= 2, "Other amount of 'ForceSubtaskResults' than 1 or 2 found for this task_id."
+    decoded_message_from_database = deserialize_message(force_subtask_results.last().data.tobytes())
+
+    if (
+        current_time > (
+            decoded_message_from_database.ack_report_computed_task.timestamp +
+            settings.SUBTASK_VERIFICATION_TIME                               +
+            settings.FORCE_ACCEPTANCE_TIME                                   +
+            settings.CONCENT_MESSAGING_TIME
```
[the remark this answers is not in the document]
Comment (rwrzesien) [reply concent_api/core/views.py]: :+1:

File: concent_api/core/views.py [hunk 2]
```diff
@@ -457,28 +457,39 @@ def handle_send_force_subtask_results_results_response(request, client_message):
...
         )
-        return HttpResponse("", status = 202)
-    else:
+    ):
         raise Http400("Time to acknowledge this task is already over.")
```
[the remark this answers is not in the document]
Comment (rwrzesien) [reply concent_api/core/views.py]: :+1:

**Comment by rwrzesien** [comment]
@cameel Updated and ready to merge.

**Comment by cameel** [comment]
Please merge it. I won't be able to do it today.

**Pull request merged**